examples/fluids/navierstokes.c

```diff
@@ -823,7 +823,6 @@ int main(int argc, char **argv) {
   PetscInt contsteps     = 0;        // -
   PetscInt degree        = 1;        // -
   PetscInt qextra        = 2;        // -
-  PetscInt degreeSur     = 1;        // -
```

Comment:
@jedbrown , why do the surface and the volume need to have the same degree?

Reply:
If you have a tensor Lagrange finite element, then the restriction to a face has the same degree. We're viewing the same set of dofs.

Reply:
Very true. Thanks.